packages/mattermost-redux/src/reducers/entities/general.ts

```diff
@@ -137,6 +138,16 @@ function firstAdminVisitMarketplaceStatus(state = false, action: GenericAction)
     }
 }
 
+function osColorScheme(state: OsColorSchemeName = 'light', action: GenericAction) {
```

Comment:
I didn't make it constant because it's typed as literal union (`OsColorSchemeName`) - it gives autocompletion in IDE and TS ensures that valid values are passed around. For me that's more handy.
However, I have no strong opinion here. I can make it as constant.
Let me know what your final decision is. I'll make code changes if needed :)